geowebcache/core/src/main/resources/org/geowebcache/config/geowebcache.xsd

```diff
@@ -137,6 +137,7 @@
             <xs:sequence>
               <xs:element ref="gwc:blobstore" minOccurs="0" maxOccurs="unbounded"/>
             </xs:sequence>
+            <xs:attribute name="id" type="xs:string"></xs:attribute>

```

Comment:
Yes it is optional, however when xstream serialises it always writes them out and there doesn't seem to be an option to turn it off for specific classes as far as I can see so far as I only really wanted to add the reference id to blobstore objects.